### PR TITLE
Reduce filebeat verbosity at INFO level

### PR DESCRIPTION
**Pull request by tsg**
Part of #1931. Replaces two of the "periodic" Infos with Debugs + expvars.

### CODE REVIEW

File: filebeat/prospector/prospector.go [hunk 1]
```diff
@@ -111,7 +111,7 @@ func (p *Prospector) Run() {
 				}
 				select {
 				case <-p.done:
-					logp.Info("Prospector channel stopped")
+					logp.Debug("prospector", "Prospector channel stopped")
```
Comment (ruflin) [inline filebeat/prospector/prospector.go]: I would probably keep these (all in this file) in Info as they show the shutdown messages (are only showed ones)

File: filebeat/prospector/prospector.go [hunk 2]
```diff
@@ -126,17 +126,17 @@ func (p *Prospector) Run() {
 	for {
 		select {
 		case <-p.done:
-			logp.Info("Prospector ticker stopped")
+			logp.Debug("prospector", "Prospector ticker stopped")
 			return
 		case <-time.After(p.config.ScanFrequency):
-			logp.Info("Run prospector")
+			logp.Debug("prospector", "Run prospector")
```
Comment (ruflin) [inline filebeat/prospector/prospector.go]: This one should stay in debug.

**Comment by tsg** [comment]
@ruflin Addressed comments.

**Comment by ruflin** [comment]
LGTM

**Pull request merged**